app/views/users/edit.html.erb

```diff
@@ -1,10 +1,6 @@
 <h1> Editing <%= @user.name %> </h1>
 <%= simple_form_for @user do |f| %>
-  <%= f.input :email %>
-  <%= f.input :first_name %>
-  <%= f.input :preferred_name %>
-  <%= f.input :middle_name %>
-  <%= f.input :last_name %>
-  <%= f.select :role, collection: User.roles.keys %>
+  <%= f.select :gender, collection: User.genders.keys %>
```

Comment:
Not that we need to change this now, but we're going to need to add some labels to these forms eventually 😛.